streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/FlinkClusterServiceImpl.java

```diff
@@ -182,21 +187,25 @@ public void start(FlinkCluster cluster) {
           String address =
               YarnUtils.getRMWebAppURL() + "/proxy/" + deployResponse.clusterId() + "/";
           flinkCluster.setAddress(address);
+          flinkCluster.setJobManagerUrl(deployResponse.address());
```

Comment:
```suggestion
```

Reply:
Here, I set the jobmanagerurl to retrieve the status of the flink cluster from the jobmanager. If you obtain the status from the resourcemanager every time, there may be too much pressure on the
resourcemanager when there are too many flink clusters, resulting in a failure to obtain the status. Previously, the status of jobs was obtained from the resourcemanager, which may lead to excessive pressure in the resourcemanager and result in a failure to obtain the status.

Reply:
got it .